install_add_pod/after/Pods/Pods.xcodeproj

```diff
@@ -218,6 +212,7 @@ Build Configurations:
       CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF: 'YES'
       CLANG_WARN_OBJC_LITERAL_CONVERSION: 'YES'
       CLANG_WARN_OBJC_ROOT_CLASS: YES_ERROR
+      CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER: 'YES'
```

Comment:
This is correct. Again we set this to `'NO'` for the pod target xcconfig. This is just the setting for the default Pods.xcodeproj.